### PR TITLE
[choreo] Temporary changes for migrating to new vhost format

### PR DESCRIPTION
**Pull request by VajiraPrabuddhaka**
### Purpose
$subject

### Issues

Fixes #

### Automation tests
 - Unit tests added: N/A
 - Integration tests added: N/A

### Tested environments

N/A

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: adapter/internal/discovery/xds/server.go [hunk 1]
```diff
@@ -802,6 +804,11 @@ func GenerateEnvoyResoucesForLabel(label string) ([]types.Resource, []types.Reso
 				}
 				clusterArray = append(clusterArray, orgIDOpenAPIClustersMap[organizationID][apiKey]...)
 				vhostToRouteArrayMap[vhost] = append(vhostToRouteArrayMap[vhost], orgIDOpenAPIRoutesMap[organizationID][apiKey]...)
+				if arrayContains(knownVhostPrefixes, strings.Split(vhost, ".")[0]) {
```
Comment (VirajSalaka) [inline adapter/internal/discovery/xds/server.go:807]: Shall we keep some sort of a variable to decide on whether we need to add two routes. If some issue occurs later, then changing it via a environment property would be the fastest.

Comment (VajiraPrabuddhaka) [reply adapter/internal/discovery/xds/server.go]: fixed via https://github.com/wso2/product-microgateway/pull/3401/commits/fc514e366269a7d18753687cdedd95c0a768afec

**Review by VirajSalaka — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20230731.4

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20230731.4

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : dev-deployment-v2 : 20230731.4

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : stage-deployment-v2 : 20230731.2

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20230731.2

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : stage-deployment-v2 : 20230731.2

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : stage-deployment-v2 : 20230731.6

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20230731.6

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20230802.2

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20230802.2

**Pull request merged**